mysql-test/columnstore/basic/r/mcs_named_udfs.result

```diff
@@ -0,0 +1,33 @@
+DROP DATABASE IF EXISTS cal_test_db;
+CREATE DATABASE cal_test_db;
+USE cal_test_db;
+create table t1(a int, b int comment 'autoincrement=1') engine=columnstore;
+select mcsflushcache();
+mcsflushcache()
+0
+select mcssettrace(0);
+mcssettrace(0)
+0
+select mcssetparms("pmmaxmemorysmallside","2048000000");
+mcssetparms("pmmaxmemorysmallside","2048000000")
+Updated pmmaxmemorysmallside 2048000000
+select mcsgettrace();
+mcsgettrace()
+NULL
+select mcsgetversion();
+mcsgetversion()
+8.1.1
```

Comment:
This result will surely change so we need to test the shape of what mcsgetversion() cal returns.

Reply:
We can use test query select calgetversion()=mcsgetversion() here